ocfweb/static/img/hosting-logos/ocf-hosted-penguin.svg

```diff
@@ -0,0 +1,292 @@
+                 inkscape:connector-curvature="0" />
+            </g>
+            <g
+               style="fill:#ffd700;fill-opacity:1"
+               id="g3222"
+               transform="translate(153.8071,561.979)">
+              <path
+                 style="fill:#ffd700;fill-opacity:1;fill-rule:nonzero;stroke:none"
+                 id="path3224"
+                 d="m 0,0 -3.438,-1.896 -3.438,9.88 2.013,1.292 z"
+                 inkscape:connector-curvature="0" />
+            </g>
+            <g
+               style="fill:#ffd700;fill-opacity:1"
+               id="g3246"
+               transform="translate(83.6748,551.3765)">
+              <path
+                 style="fill:#ffd700;fill-opacity:1;fill-rule:nonzero;stroke:none"
+                 id="path3248"
+                 d="M 0,0 7.219,3.185 8.595,-4.412 -0.688,-7.25 Z"
+                 inkscape:connector-curvature="0" />
+            </g>
+            <g
+               style="fill:#ffd700;fill-opacity:1"
+               id="g3250"
+               transform="translate(94.1646,536.2236)">
+              <path
+                 style="fill:#ffd700;fill-opacity:1;fill-rule:nonzero;stroke:none"
+                 id="path3252"
+                 d="m 0,0 h -8.104 l 0.541,4.434 9.626,3.468 z"
+                 inkscape:connector-curvature="0" />
+            </g>
+            <g
+               style="fill:#ffd700;fill-opacity:1"
+               id="g3254"
+               transform="translate(88.439,523.2188)">
+              <path
+                 style="fill:#ffd700;fill-opacity:1;fill-rule:nonzero;stroke:none"
+                 id="path3256"
+                 d="m 0,0 -9.798,-0.254 5.157,8.463 h 8.157 z"
+                 inkscape:connector-curvature="0" />
+            </g>
+            <g
+               style="fill:#ffd700;fill-opacity:1"
+               id="g3258"
+               transform="translate(82.9873,516.374)">
+              <path
+                 style="fill:#ffd700;fill-opacity:1;fill-rule:nonzero;stroke:none"
+                 id="path3260"
+                 d="m 0,0 -7.959,1.041 -5.988,-7.632 8.018,-1.918 z"
+                 inkscape:connector-curvature="0" />
+            </g>
+            <g
+               style="fill:#ffd700;fill-opacity:1"
+               id="g3262"
+               transform="translate(78.9614,542.9121)">
+              <path
+                 style="fill:#ffd700;fill-opacity:1;fill-rule:nonzero;stroke:none"
+                 id="path3264"
+                 d="M 0,0 -9.921,-6.688 -7.197,2.862 0,5.724 Z"
+                 inkscape:connector-curvature="0" />
+            </g>
+          </g>
+        </g>
+      </g>
+      <text
+         xml:space="preserve"
+         style="font-style:normal;font-weight:normal;font-size:21.1427784px;line-height:0.75;font-family:sans-serif;letter-spacing:0px;word-spacing:0px;fill:#000000;fill-opacity:1;stroke:none;stroke-width:0.52856946"
+         x="314.98428"
```

Comment:
I think the penguin and the text are too close to each other at their narrowest points.